airflow/example_dags/example_swarm_operator.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""
+from datetime import timedelta
+import airflow
+from airflow import DAG
+from airflow.contrib.operators.swarm_operator import SwarmOperator
+
+default_args = {
+    'owner': 'airflow',
+    'depends_on_past': False,
+    'start_date': airflow.utils.dates.days_ago(1),
+    'email': ['airflow@example.com'],
+    'email_on_failure': False,
+    'email_on_retry': False
+}
+
+dag = DAG(
+    'swarm_sample',
+    default_args=default_args,
+    schedule_interval=timedelta(minutes=10),
+    catchup=False
+)
+
+with dag as dag:
+    t1 = SwarmOperator(
+        api_version='auto',
+        docker_url='tcp://localhost:2375', #Set your docker URL
+        command='sleep 10',
+        image='reg-hk.agodadev.io/adp-messaging/adp-airflow:1.10.3',
```

Comment:
We should avoid hard-coded  specific constants I think - can we read it from an environment variable (see GCP operators) and keep some generic name here?

Reply:
Ohh no.. that was not supposed to be there. 🙈 
I couldn't figure out which GCP operator to look (and what to look for) but I can do something like this https://github.com/apache/airflow/blob/master/airflow/example_dags/example_docker_operator.py#L53